db/schema.rb

```diff
@@ -0,0 +1,61 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_02_07_161819) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.text "Text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "AuthorId", null: false
+    t.bigint "PostId", null: false
+    t.index ["AuthorId"], name: "index_comments_on_AuthorId"
+    t.index ["PostId"], name: "index_comments_on_PostId"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "PostId", null: false
+    t.bigint "AuthorId", null: false
+    t.index ["AuthorId"], name: "index_likes_on_AuthorId"
+    t.index ["PostId"], name: "index_likes_on_PostId"
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.string "Title"
+    t.text "Text"
+    t.integer "CommentsCounter"
+    t.integer "LikesCounter"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "AuthorId", null: false
+    t.index ["AuthorId"], name: "index_posts_on_AuthorId"
```

Comment:
- [ ] Kindly change `Text`, `CommentsCounter`,` LikesCounter`, `AuthorId` & `PostId` to text, comments_counter, likes_counter, author_id.